### PR TITLE
default to nlcd data from quilt

### PR DESCRIPTION
**Pull request by knaaptime**

### CODE REVIEW

**Comment by knaaptime** [comment]
aside from reformatting, this does a few things:
  - checks whether there are both extensive and intensive variables to interpolate  
  - adds [this](https://github.com/pysal/tobler/blob/cf0f2839ec7b63a9bc05cb99a74ba4db32b245d0/tobler/data.py) file that include some nlcd utilities
  - wraps calles to raster_path inside `fetch_quilt_path` so that you can use nlcd easily

**Comment by knaaptime** [comment]
and you can see these changes in action in [this](https://github.com/spatialucr/geosnap/blob/a55ea6342f0dec00a33ff2bcfa19fbecfae63bc8/examples/03_harmonizing_community_boundaries.ipynb) notebook

File: tobler/data.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    """
+    quilt3.Package.install("rasters/nlcd", "s3://quilt-cgs")
+
+
+def fetch_quilt_path(path):
+    """utility for getting the path to a raster stored with quilt.
+
+    Parameters
+    ----------
+    path : str
```
Comment (renanxcortes) [inline tobler/data.py]: Shouldn't be better to inform here what are the possible options of the function (i.e. ["nlcd_2011", "nlcd_2001"])?

Comment (knaaptime) [reply tobler/data.py]: good idea. this also reminds me there's a 2016 vintage i need to convert/upload to quilt

Comment (knaaptime) [reply tobler/data.py]: i added a more thorough description. Are you ok with merging so i can test whether i configured https://github.com/spatialucr/geosnap/pull/126 properly?

**Pull request merged**